package.json

```diff
@@ -1,104 +1,64 @@
-    "cov": "jest --coverage=true && yarn cov-open",
-    "cov-open": "open-cli coverage/lcov-report/index.html",
-    "docs": "yarn docs:html && open-cli docs/index.html",
-    "docs:test": "yarn docs:html",
-    "docs:html": "typedoc src/index.ts --excludePrivate --theme default --out docs",
-    "docs:json": "typedoc --json docs/typedoc.json src/index.ts",
-    "release": "standard-version",
-    "release:bump-release": "yarn release",
-    "release:bump-prerelease": "yarn release --prerelease",
-    "reset": "git clean -dfx && git reset --hard && yarn",
-    "validate:dependencies": "yarn audit --groups dependencies && yarn license-validate",
-    "validate:dev-dependencies": "yarn audit --groups devDependencies",
-    "license-validate": "yarn sofie-licensecheck --allowPackages caniuse-lite@1.0.30001373"
-  },
-  "engines": {
-    "node": ">=14"
-  },
-  "files": [
-    "/dist",
-    "/CHANGELOG.md",
-    "/README.md",
-    "/LICENSE"
-  ],
-  "devDependencies": {
-    "@sofie-automation/code-standard-preset": "^2.0.2",
-    "@types/jest": "^27.5.2",
-    "@types/node": "^14.18.23",
-    "jest": "^27.2.1",
-    "mockdate": "^3.0.5",
-    "open-cli": "^7.0.1",
-    "rimraf": "^3.0.2",
-    "standard-version": "^9.3.1",
-    "ts-jest": "^27.0.5",
-    "typedoc": "^0.22.18",
-    "typescript": "~4.5"
-  },
-  "keywords": [
-    "mos",
-    "broadcast",
-    "socket",
-    "typescript",
-    "javascript",
-    "open",
-    "source",
-    "automation",
-    "ncs",
-    "nrcs",
-    "rundown",
-    "production"
-  ],
-  "dependencies": {
-    "iconv-lite": "^0.6.2",
-    "moment": "^2.29.4",
-    "tslib": "^2.2.0",
-    "xml-js": "^1.6.11",
-    "xmlbuilder": "^15.1.1"
-  },
-  "standard-version": {
-    "message": "chore(release): %s",
-    "tagPrefix": ""
-  },
-  "prettier": "@sofie-automation/code-standard-preset/.prettierrc.json"
+	"name": "mos-connection-monorepo",
+	"version": "0.0.0",
+	"repository": "https://github.com/nrkno/tv-automation-package-manager",
+	"private": true,
+	"workspaces": [
+		"packages/*"
+	],
+	"scripts": {
+		"lerna": "lerna",
+		"postinstall": "node scripts/update-packages.js",
+		"ci": "yarn install && yarn build && yarn lint && yarn test",
+		"release:set-version": "lerna version --exact --no-changelog --no-push --yes",
+		"release:bump-release": "lerna version --exact --conventional-commits --conventional-graduate --tag-version-prefix='' --no-push --force-publish=*",
+		"release:bump-prerelease": "lerna version --exact --conventional-commits --conventional-prerelease --tag-version-prefix='' --no-push --force-publish=*",
+		"set-version": "lerna version --exact --no-changelog --no-git-tag-version --no-push --yes",
+		"setup": "lerna bootstrap",
+		"reset": "node scripts/reset.js",
+		"link:all": "lerna exec -- yarn link",
+		"build": "lerna run build --stream",
+		"build:changed": "lerna run build --since head --exclude-dependents --stream",
+		"lint": "lerna exec --parallel --no-bail -- eslint . --ext .ts,.tsx",
```

Comment:
This is returning with 'success' when it fails in one of the packages fails.
Resulting in the ci passing when the linter has errors (there are a couple right now)

Reply:
Good catch! Removing `--no-bail` fixes the issue